recipe/build.sh

```diff
@@ -3,12 +3,24 @@
 set -exuo pipefail
 
 mkdir -p build-target
+
 pushd build-target
-  cmake ${CMAKE_ARGS} \
-    -GNinja \
-    -DCMAKE_BUILD_TYPE:STRING="Release" \
-    -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON \
-    -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
-    "${SRC_DIR}"
-  ninja
+if [[ ${cuda_compiler_version} != "None" ]]; then
+    cmake ${CMAKE_ARGS} \
```

Comment:
Is there any difference between these two CMake calls?

Reply:
The second one adds `DUSE_CUDA=ON` which is .. backwards. Ooops, will fix it!

Reply:
Can you instead use the following? 
```
if [[ ${cuda_compiler_version} != "None" ]]; then
  export CMAKE_ARGS="-DUSE_CUDA=ON ${CMAKE_ARGS}"
fi
```

That would duplicate less code.